bindata/network/openshift-sdn/sdn-ovs.yaml

```diff
@@ -139,33 +139,32 @@ spec:
         livenessProbe:
           exec:
             command:
-            # Validate that ovs-vswitchd and ovsdb are running. These three commands check
-            # 1. that the pids in the pid file are up
-            # 2. that ovs-vswitchd is responding to queries
-            # 3. that ovsdb is responding to queries
-            # 4. if br0 is configured, that its management process is responding
+            # Validate that ovs-vswitchd and ovsdb are running. Check that
+            # 1. that ovs-vswitchd is responding to queries
+            # 2. that ovsdb is responding to queries
+            #
+            # Need the manual target file because ovs-appctl doesn't like
+            # being in a different namespace from ovs-vswitchd.
```

Comment:
If OVS is running in systemd we shouldn't be liveness-probing it anyway... I'd just do

    if [ -f /host/var/run/ovs-config-executed ]; then
      exit 0
    fi

for both probes

Reply:
Right, I thought about that, but I want to halt rollouts if OVS isn't running, regardless of who should be running it.

Reply:
But ovs should always be running (systemd has the unit set to "Restart=Always") if its running under systemd. I think that is what @danwinship was referring to.

Reply:
And, as we've found, there are lots of cases where we got this wrong. The readiness probe protects us from taking down the cluster if we miss yet another corner case.

I really think we should keep the readiness probe.

Reply:
We should certainly remove the liveness probe anyway

Reply:
(if not the readiness probe)